src/lib/constants/index.js

```diff
@@ -56,3 +56,9 @@ export const MEDIATYPE_DOCKER_MANIFEST_V2 =
 export const RELEASE_NAME_PATTERN = /^(?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63}$/g;
 export const RELEASE_NAME_AIRFLOW_PATTERN = /^\/((?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63})\/(airflow|flower)/;
 export const VALID_DOCKER_IMAGE_NAME = /^(?<releaseName>(?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63})\/airflow$/;
+
+// NATS related constants
+
+export const DEPLOYMENT_IMAGE_UPDATE = "houston.deployment.image.update";
+export const DEPLOYMENT_IMAGE_UPDATE_ID = "deployment-image-update-worker";
```

Comment:
```suggestion
export const DEPLOYMENT_IMAGE_UPDATE = "houston.deployment.image.update";
export const DEPLOYMENT_IMAGE_UPDATE_ID = "houston-deployment-image-update-worker";
```

Another nitpick, but these are almost the same, just slightly different ha